pyproject.toml

```diff
@@ -41,7 +41,7 @@ pyink-indentation = 2
 pyink-use-majority-quotes = true
 
 [build-system]
-requires = ["setuptools>=61", "numpy~=1.21", "pybind11>=2.10.3"]
+requires = ["setuptools>=61", "numpy~=1.21.0", "pybind11>=2.10.3"]
```

Comment:
Should we pin the other deps also?

Reply:
I think these are not as important as getting numpy right, because there's no API compatibility concern.

Reply:
Sure. I just figured there was no harm in being more aggressive with our pins to prevent breakage on new releases.